### PR TITLE
Update extension for Thunderbird 68.*

### LINKED ISSUE #25: Project inactive? / Thunderbird 68 compatibility
Title.

### PR DESCRIPTION
**Pull request by m-rey**
Fix #25.
Sometimes it is easy to port a Overlay Extension to a MailExtension
simply by reformatting the install.rdf file to a manifest.json file.
(see https://developer.thunderbird.net/add-ons/updates/overlays)

-------------------------------------------------------------------

- Use the code from https://github.com/gjedeer/paranoia/issues/25#issuecomment-533324784
  as a basis.
- Remove the options key as indicated in the comment https://github.com/gjedeer/paranoia/issues/25#issuecomment-533494277
- Add author.
- Add project website.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "tls-paranoia@gdr.name",
+      "strict_min_version": "68.0"
+    }
+  },
+  "name": "Paranoia",
+  "description": "Check if your emails arrived encrypted (and which corporations were able to read them nevertheless)",
+  "version": "0.2.7",
```
Comment (sdellenb) [inline manifest.json]: I think it needs a version bump for uploading to https://addons.thunderbird.net/en-US/thunderbird/addon/paranoia/

```suggestion
  "version": "0.3.0",
```

Comment (m-rey) [reply manifest.json]: done

**Comment by gjedeer** [comment]
Nice! I'll review/test/release on the weekend.

**Comment by sdellenb** [comment]
I'm currently testing it and it loads fine.
As I could not get a sad smiley (for testing), I recognized that I also hit #24 and everything gets a pass.

**Comment by gjedeer** [comment]
Woah, there are so much fewer sad faces than the last time I tested this extension, the state of SMTP really has improved. Unless I'm hitting Issue #24 ;)

Anyway. I'm trying to submit it to addons.mozilla.org, and I even made it through the recaptcha which ironically doesn't work in Firefox (but recaptcha works for me on all other pages...), but it seems that I can only choose that this addon is for Firefox or Firefox Mobile, Thunderbird has disappeared.

I've submitted it anyway but I'm a bit afraid it's not going to appear in the Thunderbird addons manager and instead I'm going to get backslash from angry Firefox users saying it's not working. Anyone here familiar with this?

**Comment by gjedeer** [comment]
Okay, found https://addons.thunderbird.net and submitted it there.

`Queue Position: 27 of 27`

**Pull request merged**